src/data/pins.ts

```diff
@@ -59,6 +68,10 @@ const plannableSlimes: Resource[] = [
     { name: "Tangle", type: "Docile" },
     { name: "Tarr", type: "Hostile" },
     yolkySlime,
+    { name: "Sloomber", type: "Docile" },
```

Comment:
Can't have this alphabetical, or we'll break existing plans that are loaded from local storage.